apps/editorial_features/components/eoy/client.coffee

```diff
@@ -28,57 +31,63 @@ module.exports.EoyView = class EoyView extends Backbone.View
 
   watchWindow: =>
     $(window).scroll () =>
-      @trackDirection()
+      throttled = _.throttle(@trackDirection, 100)
+      if window.scrollY != @windowPosition
+        throttled()
     $(window).resize () =>
       @setupSliderHeight()
+      @windowHeight = $(window).height()
       $.waypoints('refresh')
 
   getScrollZones: =>
     scrollZones = []
-    scrollZones.push @firstHeight
+    scrollZones.push @containerHeight
     for i in [1..($('.scroller__items section').length - 1)]
-      scrollZones.push( (i * @activeHeight) + @firstHeight )
+      scrollZones.push( (i * @activeHeight) + @containerHeight )
     return scrollZones
 
-  closestSection: (scrollTop) =>
-    scrollZones = @getScrollZones()
+  closestSection: (scrollTop, scrollZones) =>
     closest = Math.max.apply(null, scrollZones)
     for i in [0..scrollZones.length + 1]
       if scrollZones[i] >= scrollTop and scrollZones[i] < closest
         closest = i
     return closest
 
   trackDirection: =>
-    scrollTop = $(window).scrollTop()
+    scrollTop = window.scrollY
+    scrollTop = Math.round(scrollTop)
     if scrollTop == 0
       $('.scroller__items section[data-section!="0"]').attr('data-state', 'closed')
-      $('.scroller__items section[data-section="0"]').attr('data-state', 'open').height(@firstHeight)
-    #else if scrollTop > @windowHeight
+      $('.scroller__items section[data-section="0"]').attr('data-state', 'open').height(@containerHeight)
+    #else if scrollTop > @windowPosition
       #downscrolling
     #else
       #upscrolling
     @doSlider(scrollTop)
     if scrollTop >= @getScrollZones()[6]
+      # throttled = _.throttle(@animateBody, 100)
```

Comment:
Would it be ok to remove this commented out bits of code? I see a few in the Stylus sheets as well.